### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by hone1er**
### What changes were made?

- On login user is sent to dashboard instead of profile
- Profile has been removed from UI
- Marketplace has been removed from UI
- Stripe integration added

### Is there any background info about the change?

- Start of Project Lincoln 
- User can create and view stripe connect account from dashboard(will be moved to profile edit page later)
- Through the SOW builder the user can create and send an invoice 
- Payment can be made through the project page
- Project page will display a receipt after payment is made

### Are there any state changes? Mention key ones (if any)

userState.stripeAccountId: string

### Any new dependencies? Why were they added?

- stripe
- @stripe/react-stripe-js
- @stripe/stripe-js

### CODE REVIEW

**Review by NickGonzalez04 — CHANGES_REQUESTED** [review]
Not a change necessarily if it is intentional. Just want to double-check first!

File: pages/api/stripe/charges/createInvoice.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import { NextApiRequest, NextApiResponse } from "next";
+
+export default async function createChargeHandler(
```
[the remark this answers is not in the document]
Comment (hone1er) [reply pages/api/stripe/charges/createInvoice.ts:3]: Yeah it shouldn't make a difference but i'll change that in the branch i'm working on to createInvoiceHandler

**Review by nagmak — APPROVED** [review]
Looks good!

File: pages/api/stripe/charges/createInvoice.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (NickGonzalez04) [reply pages/api/stripe/charges/createInvoice.ts:3]: 💯

**Review by NickGonzalez04 — APPROVED** [review]

**Pull request merged**